### DISCUSSION TITLE
Possible to place and reference presets in nested directories?

### DISCUSSION THREAD

**Discussion by sppatel** [renovatebot/renovate#8396] [Request Help] [answered]
![Screen Shot 2021-01-22 at 1 27 40 PM](https://user-images.githubusercontent.com/989367/105534948-38b62400-5cbc-11eb-8ba6-c63a4efc5fad.png)

The above screenshot refers to being able to reference a preset via `abc/foo:xyz` which resolves to the file xyz.json in the `github.com/abc/foo` repository.   However, am I able to reference a directory structure such that

`abc/foo:someDir/xyz` resolves to the file in `someDir/xyz.json` in the same repo?

Goal: I want to organize the presets I have - and not have them all flattened to the root of the repo.

**Comment by rarkins** [comment, marked as answer]
Not supported currently
